servicetalk-loadbalancer-experimental/src/main/java/io/servicetalk/loadbalancer/DefaultLoadBalancerBuilder.java

```diff
@@ -40,7 +40,7 @@ final class DefaultLoadBalancerBuilder<ResolvedAddress, C extends LoadBalancedCo
     @Nullable
     private LoadBalancerObserverFactory loadBalancerObserverFactory;
     private LoadBalancingPolicy<ResolvedAddress, C> loadBalancingPolicy = defaultLoadBalancingPolicy();
-    private ConnectionPoolPolicy<C> connectionPoolPolicy = defaultConnectionSelectorFactory();
+    private ConnectionSelectorPolicy<C> connectionSelectorPolicy = defaultConnectionSelectorFactory();
```

Comment:
Do we need to rename `defaultConnectionSelectorFactory()` -> `defaultConnectionSelectorPolicy()`?

Reply:
Good call.